### PR TITLE
position search-bar properly according to window

### PR DESCRIPTION
**Pull request by chovy**
refs https://github.com/imusify/imusify-web/issues/124

### CODE REVIEW

File: src/components/SearchBar.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <form class="search-bar"
         @submit.prevent="search()">
     <input type="text"
-           placeholder="What are you looking for?"
+           placeholder="Search"
```
Comment (iamonuwa) [inline src/components/SearchBar.vue]: Can we increase the size of the search bar? Every other thing looks good to me

Comment (chovy) [reply src/components/SearchBar.vue]: It needs to fit on mobile and this "What are you looking for?" is too big.

Comment (iamonuwa) [reply src/components/SearchBar.vue]: Can we make it overlay on click? So that we can have autocomplete for the search

Comment (iamonuwa) [reply src/components/SearchBar.vue]: Will fit in on mobile and desktop better

Comment (chovy) [reply src/components/SearchBar.vue]: i dont' think we should overlay the menu because then you can't click the "next" button for categories.

**Pull request merged**